### ISSUE TITLE
Getting low auc while training on UCF-Crime Data using I3D features(10-crop)

### ISSUE THREAD

**Issue by Shashank56** [tianyu0207/RTFM#13]
We have used the following repo for extracting 10-crop I3D features.
https://github.com/Tushar-N/pytorch-resnet3d

But while training we are not having any improvement in the auc after 400 iterations. I have attached the graph shown in visdom, so is there anything that we are doing wrong? And when will you publish the I3D features for UCF-Crime dataset?
![graph](https://user-images.githubusercontent.com/35572297/116369859-77adc580-a827-11eb-8836-7b3129db44cb.jpg)

**Comment by Shashank56** [comment]
Can you please post the exact I3D code that you have used? We have used the above repo as mentioned and made some changes to incorporate 10crop and rgb frames. Here is the link for the code. 
https://github.com/GowthamGottimukkala/I3D_Feature_Extraction_resnet
But we couldn't get the results that we have expected. It would be helpful if you send your exact code.

**Comment by tianyu0207** [comment]
> Can you please post the exact I3D code that you have used? We have used the above repo as mentioned and made some changes to incorporate 10crop and rgb frames. Here is the link for the code.
> https://github.com/GowthamGottimukkala/I3D_Feature_Extraction_resnet
> But we couldn't get the results that we have expected. It would be helpful if you send your exact code.

I have uploaded the UCF features and checkpoint. Please change the video level labels accordingly in the dataloader when using the UCF. 

Cheers

**Comment by nathanb97** [comment]
Hey, I took a look at your code.
Do you end up getting good results?
If yes, did you do a normalisation ?
